Source/JobGiver_ZombieResponse.cs

```diff
@@ -13,87 +13,61 @@ public class JobGiver_ZombieResponse : ThinkNode_JobGiver
         // Token: 0x060003E6 RID: 998 RVA: 0x0002923C File Offset: 0x0002763C
         protected override Job TryGiveJob(Pawn pawn)
         {
-            return this.TryGetAttackNearbyEnemyJob(pawn);
+            Pawn_Zombiefied predator = pawn as Pawn_Zombiefied;
+            Pawn prey = BestPawnToHuntForPredator(predator);
+            if (prey == null) return null;
+            return new Job(ZombiefiedMod.zombieHunt, prey);
         }
 
-        // Token: 0x060003E7 RID: 999 RVA: 0x000292AC File Offset: 0x000276AC
-        private Job TryGetAttackNearbyEnemyJob(Pawn pawn)
-        {
-            Pawn thing = BestPawnToHuntForPredator(pawn);
-            if (thing != null)
-            {
-                return new Job(ZombiefiedMod.zombieHunt, thing)
-                {
-                    killIncappedTarget = true,
-                    expiryInterval = (int)(Rand.RangeSeeded(1f, 2f, Find.TickManager.TicksAbs) * 700),
```

Comment:
expiryInterval is important because without it zombies will follow a pawn forever and never lose sight and they shouldnt do that

Reply:
oh man I just stumbled upon the usage in jobs in the decompilation, no idea why it wasn't showing up before. blaaah. Anyway I went through and tested it and yeah definitely important. Gonna add it in.